earth_enterprise/src/fusion/autoingest/sysman/gefusion

```diff
@@ -58,17 +58,34 @@ function check_if_master {
 }
 
 function stop_daemon {
-    echo -n "Shutting down $2... "
-    pidfile="/var/opt/google/run/$1.pid"
-    if [ -e $pidfile ] ; then
-	if gestopdaemon $1 ; then
-	    echo "Done"
-	else
-	    echo "Failed"
-	fi
+  echo -n "Shutting down $2... "
+  pidfile="/var/opt/google/run/$1.pid"
+  if [ -e $pidfile ] ; then
+    if gestopdaemon $1 ; then
+      echo "Done"
     else
-	echo "Not running"
+      echo "Failed"
     fi
+  else
+    echo "Not running"
+  fi
+}
+
+function check_daemon {
+  # more DRY to use "gefdaemoncheck --checkrunning"
+  #   but it seems to assume --checksysman is true
+  #   thus wont work on non-master hosts.
+  daemon_path=$(which "$1")
+  pidfile="/var/opt/google/run/$1.pid"
+  if [ -s $pidfile ] ; then
```

Comment:
Lint: Please, comment variable expansion on `if` line and below when using `cat`.

Reply:
Fixed, thanks.